app/styles/partials/text.scss

```diff
@@ -23,3 +23,7 @@
   text-overflow: ellipsis;
   white-space: nowrap;
 }
+
+.weight-600 {
```

Comment:
That's not good, weight class should not dictate color of text

Reply:
But sir, in divyam's PR you told to do this.

Reply:
I told to change the color from whichever class adds it. I don't think weight class adds it